### PR TITLE
Remove default spacing of SideMenu (inherited from HorizontalLayout)

### PR DESCRIPTION
**Pull request by pbaris**
This change is [Reviewable](https://reviewable.io/reviews/tsuoanttila/sidemenu-addon/10)

### CODE REVIEW

File: sidemenu-addon/src/main/java/org/vaadin/teemusa/sidemenu/SideMenu.java [hunk 1]
```diff
@@ -111,6 +108,9 @@ public void buttonClick(final ClickEvent event) {
 		menuItemsLayout.addStyleName("valo-menuitems");
 		menuArea.addComponent(menuItemsLayout);
 
+		if (contentArea instanceof VerticalLayout) {
+			((VerticalLayout) contentArea).setMargin(false);
```
Comment (tsuoanttila) [inline sidemenu-addon/src/main/java/org/vaadin/teemusa/sidemenu/SideMenu.java]: I think I already made this change in the master branch. 

https://github.com/tsuoanttila/sidemenu-addon/blob/master/sidemenu-addon/src/main/java/org/vaadin/teemusa/sidemenu/SideMenu.java#L161

Comment (pbaris) [inline sidemenu-addon/src/main/java/org/vaadin/teemusa/sidemenu/SideMenu.java]: Yes you are right. I will fix it

Comment (pbaris) [inline sidemenu-addon/src/main/java/org/vaadin/teemusa/sidemenu/SideMenu.java]: I have already removed it in the next commit

**Comment by tsuoanttila** [comment]
:lgtm:

---

Reviewed 4 of 4 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/tsuoanttila/sidemenu-addon/10#-:-Km69QXI6cQX99J5fNnF:bnfp4nl)*

**Pull request merged**